python/delta/connect/tests/test_deltatable.py

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (2024) The Delta Lake Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from delta.connect.testing.utils import DeltaTestCase
+import os
+import sys
+
```

Comment:
Ran into this issue (https://stackoverflow.com/questions/4383571/importing-files-from-different-folder)